<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>90</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string> Сформировать</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>391</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Вставьте путь к актуальному шаблону</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>431</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>171</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Грязная маржа:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Чистая маржа:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Кол-во рейсов:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Кол-во срывов:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Итог Зарплата:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>149</y>
      <width>371</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="gr_m">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ch_m">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="count_r">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="count_s">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="salary">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>241</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Все</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>январь</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>февраль</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>март</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>апрель</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>май</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>июнь</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>июль</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>август</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>сентябрь</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>октябрь</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ноябрь</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>декабрь</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>comboBox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>pushButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>548</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>558</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
